public/locales/en/modules/torrents-status.json

```diff
@@ -63,7 +63,8 @@
       }
     },
     "loading": {
-      "title": "Loading..."
+      "title": "Loading...",
+      "description": "Homarr is establishing a connection..."
```

Comment:
Same thing, don't say "homarr" in translations. Just "establishing connection"

Reply:
The App name can be replaced with a variable